includes/attendee/attendee-repository.php

```diff
@@ -114,6 +115,43 @@ public function get_attendees( int $event_id ): array { // phpcs:ignore Generic.
 		return array();
 	}
 
+	/**
+	 * Get the hosts' users for an event.
+	 *
+	 * @param int $event_id The id of the event.
+	 * @return array[WP_User] The hosts of the event.
+	 */
+	public function get_hosts( int $event_id ): array {
```

Comment:
I think this method should return an array of `Attendee`. The idea with the repositories is that they always "deal with" a specific entity.

For example in the `Event_Repository`, methods that modify an event take an `Event` instance, and methods that return events return instances of `Event`.

I think it would make sense to continue to follow that pattern here, so this method would return an array of `Attendee`. Then you would call the method with something like:

```php
$host_attendees = $attendee_repository->get_hosts( $event_id );

// If you would need an instance of WP_user, you would:
$host = new WP_User( $host->attendeees[0] );
```